doc/classes/AnimatedSprite.xml

```diff
@@ -24,14 +24,14 @@
 			<argument index="1" name="backwards" type="bool" default="false">
 			</argument>
 			<description>
-				Play the animation set in parameter. If no parameter is provided, the current animation is played. Property [code]backwards[/code] plays the animation in reverse if set to [code]true[/code].
+				Plays the animation set in parameter. If no parameter is provided, the current animation is played. Property [code]backwards[/code] plays the animation in reverse if set to [code]true[/code].
```

Comment:
This line is kind of awkward... Maybe it could be phrased like this:

"Plays the animation **named [code]anim[/code]**. If no **[code]anim[/code]** is provided[...]"

What do you think?